policyengine_us/variables/gov/states/oh/tax/income/credits/joint_filing_credit/oh_joint_filing_credit_eligible.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class oh_joint_filing_credit_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for the Ohio Joint Filing Credit"
+    definition_period = YEAR
+    reference = "https://codes.ohio.gov/ohio-revised-code/section-5747.05"
+    defined_for = StateCode.OH
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
+        person = tax_unit.members
+        is_head = person("is_tax_unit_head", period)
+        is_spouse = person("is_tax_unit_spouse", period)
+        p = parameters(
+            period
+        ).gov.states.oh.tax.income.credits.joint_filing_credit
+        income = person("employment_income", period)
```

Comment:
does it also include self-employment income?

Reply:
need to recreate 
oh_agi -> oh_alternative_agi
irs_gross_income -> oh_irs_gross_income
adjusted_gross_income -> oh_adjusted_gross_income

in the irs_gross_income we want to use a for loop for the income sources to exclude the income types as said on the form